Library/Homebrew/test/language/node_spec.rb

```diff
@@ -22,23 +22,6 @@
     end
   end
 
-  describe "#std_npm_install_args" do
-    npm_install_arg = "libexec"
-
-    it "raises error with non zero exitstatus" do
-      expect { subject.std_npm_install_args(npm_install_arg) }.to \
-        raise_error("npm failed to pack #{Dir.pwd}")
-    end
-
-    it "does not raise error with a zero exitstatus" do
-      allow(Utils).to receive(:popen_read).with("npm pack").and_return("pack")
```

Comment:
How do I mock up 
```
output = `#{pack_cmd}`
```
similar to `output = Utils.popen_read("npm pack").chomp` was mocked up previously?